apisix/plugins/http-logger.lua

```diff
@@ -70,6 +80,11 @@ local _M = {
 
 
 function _M.check_schema(conf, schema_type)
+    local ok, err = log_util.check_log_scheme(conf)
```

Comment:
We should not check log schema with the TYPE_METADATA schema_type. And it is schema, not scheme.